.github/workflows/release.yml

```diff
@@ -77,10 +72,8 @@ jobs:
       - name: Create Exe Installer
         run: ./gradlew buildExePackage
```

Comment:
We should pass the `version` to this command as well, see https://stackoverflow.com/a/40239092